### PR TITLE
python3Packages.pyairvisual: 5.0.4 -> 5.0.5

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 5.0.5

Change log: https://github.com/bachya/pyairvisual/releases/tag/5.0.5

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pyairvisual/default.nix [hunk 1]
```diff
@@ -43,13 +43,14 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
-  pytestFlagsArray = [
-    "tests"
-  ];
+  pytestFlagsArray = [ "tests" ];
```
Comment (mweinelt) [inline pkgs/development/python-modules/pyairvisual/default.nix:46]: Odd, doesn't pytestCheckHook pick up that directory by default?

**Comment by mweinelt** [comment]
>  python3Packages.pyairvisual: update attributes 

This would be the meta attribute set, so `update meta` would be more fitting.

File: pkgs/development/python-modules/pyairvisual/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/pyairvisual/default.nix]: It's more that I forgot zu update it.

**Comment by fabaff** [comment]
> This would be the meta attribute set, so update meta would be more fitting.

Updated

File: pkgs/development/python-modules/pyairvisual/default.nix [hunk 2]
```diff
@@ -24,7 +24,7 @@ buildPythonPackage rec {
     owner = "bachya";
     repo = pname;
     rev = version;
-    sha256 = "0z769xrb6w6bhqcq02sjryl1qyvk9dc1xfn06fc3mdqnrbr0xxj3";
+    sha256 = "0jjvng3py5g97gvx6rdbk5zxbn5rw8gq1ki4qi4vfsypchxbpz2q";
   };
 
   nativeBuildInputs = [ poetry ];
```
Comment (mweinelt) [inline pkgs/development/python-modules/pyairvisual/default.nix:30]: Please create an upstream issue to migrate the build-backend to `poetry-core`. See https://github.com/NixOS/nixpkgs/issues/103325.

**Comment by mweinelt** [comment]
Result of `nixpkgs-review pr 108677` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python37Packages.pyairvisual</li>
    <li>python38Packages.pyairvisual</li>
  </ul>
</details>

**Review by mweinelt — APPROVED** [review]

File: pkgs/development/python-modules/pyairvisual/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/development/python-modules/pyairvisual/default.nix:30]: Done

https://github.com/bachya/pyairvisual/issues/69

**Pull request merged**